extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -233,6 +290,14 @@ oppia.directive('imageWithRegionsEditor', [
           }
           $scope.userIsCurrentlyDrawing = false;
           $scope.userIsCurrentlyDragging = false;
+          $scope.userIsCurrentlyResizing = false;
+          if ($scope.outOfRegion) {
+            $scope.xDirection = 0;
+            $scope.yDirection = 0;
+          }
```

Comment:
Actually, sorry--I retract what I said earlier. After thinking more about it I think the current behavior is okay.

But now I'm wondering why can't you just use xDirectionToggled and yDirectionToggled instead of  having to use outOfRegion?